rai/node/node.cpp

```diff
@@ -2834,6 +2872,9 @@ void rai::active_transactions::announce_votes ()
 				i->election->confirm_cutoff (transaction);
 				auto root_l (i->election->votes.id);
 				inactive.push_back (root_l);
+				election_result result;
+				election_l->get_progress(transaction, result);
+				add_election_history(root_l, result);
```

Comment:
Note to reviewer: We could only track this for explicitly requested blocks, but then it becomes purely an appendage for the reconfirmation feature vs a more general recent election history system. Easy to change, either way.